src/cluster/setup.rst

```diff
@@ -211,12 +215,11 @@ requires all other nodes to be able to see it and vice versa.
 Set up will not work with unavailable nodes.
 The notion of "setup coordination node" will be gone once the setup is finished.
 From then on, the cluster will no longer have a "setup coordination node".
-To add a node run these two commands:
+To add a node run this commands for each node you want to add:
 
 .. code-block:: bash
 
-    curl -X POST -H "Content-Type: application/json" http://admin:password@127.0.0.1:5984/_cluster_setup -d '{"action": "enable_cluster", "bind_address":"0.0.0.0", "username": "admin", "password":"password", "port": 15984, "node_count": "3", "remote_node": "<remote-node-ip>", "remote_current_user": "<remote-node-username>", "remote_current_password": "<remote-node-password>" }'
```

Comment:
No, actually, you need this first statement too. This sets the bind address, bind port, admin username and admin password on the remote node. The node_count parameter is also critical.

You can see a test cluster being set up in the test case for the `couch_setup` application [here](https://github.com/apache/couchdb-setup/blob/master/test/t-frontend-setup.sh) and [here](https://github.com/apache/couchdb-setup/blob/master/test/t.sh).